client/src/components/auth/SignUp.vue

```diff
@@ -81,6 +81,14 @@
                     {{ lang.SIGN_UP }}
                 </VBtn>
             </VForm>
+            <VBtn
+                height="44"
+                depressed
+                class="signup-button  primary"
+                @click="onSignUp"
+            >
+                {{ lang.SIGN_UP }}
+            </VBtn>
```

Comment:
Do we really need this? I see a similar button a few lines above.

Reply:
No, we don't.